coops-ui/src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -19,28 +19,28 @@
                 <v-col cols="6" class="v-col-padding">
                   <div class="list-item__title mb-1">{{item.name}}</div>
                 </v-col>
-                <v-col cols="6" class="v-col-padding" v-if="isCoaFutureEffective(item.name, item.status)">
-                  <v-scale-transition>
-                    <v-tooltip
-                      top
-                      content-class="pending-tooltip"
-                    >
-                      <template v-slot:activator="{ on }">
-                        <div id="pending-alert" class="list-item__subtitle" v-on="on">
-                          <v-icon color="yellow" small>mdi-alert</v-icon>
-                          PAID AND PENDING
-                        </div>
-                      </template>
-                      <span>
+              </v-row>
+              <div class="list-item__subtitle">
+                <v-scale-transition v-if="isCoaFutureEffective(item.name, item.status)">
```

Comment:
Can you really put a v-if in a transition? I thought transitions were only wrappers for elements that changed visibility or existence.

(Sorry, can't check online docs as they're currently broken.)

Reply:
It works as expected but i will 100% dig into it further and look at the options!

Reply:
Thanks for looking into it and letting me know!